roles/include_components/README.md

```diff
@@ -1,15 +1,42 @@
-# include_components role
+# redhat.ocp.include_components role - create and attach DCI components to the DCI jobs
 
-The `include_components` role adds components in the DCI jobs from software resources available in the ansible controller node, for example: rpm packages, and git repositories used during the job execution.
+# Synopsis
+
+The `include_components` role creates and attaches DCI components to the DCI jobs by generating them from provided resources. Possible resource inputs include RPM packages, Git repositories, and Github commits.
 
 Follow this post to know more about [components](https://blog.distributed-ci.io/automate-dci-components.html).
 
-## Configuration
+# Parameters
 
 Variables used by this role:
 
 | Setting        | Required | Type   | Description                                                        |
 | -------------- | -------- | ------ | -------------------------------------------------------------------|
-| ic\_rpms       | True     | List   | List of RPM names to include as components                         |
-| ic\_gits       | True     | List   | List of directories from GIT repositories to include as components |
-| ic\_dev\_gits  | True     | List   | List of complimentary directories from GIT repositories to include as components |
+| ic_rpms        | False    | List   | Optional. List of RPM names to include as components                         |
```

Comment:
Don't you need to escape the underscores anymore?

Reply:
it's displayed fine and the search over the repo works better, so I removed this one